packages/async-loader/async.js

```diff
@@ -29,7 +38,13 @@ export default function(req) {
 			return !nxt;
 		};
 
-		this.render = (p, s) => (s.child ? h(s.child, p) : old);
+		this.render = (p, s) => {
+			if (!s.urlData || s.urlData.url !== window.location.pathname) {
```

Comment:
This doesn't match `/profile/john` to `/profile/john/` which `preact-router` does.
@developit is there an export from `preact-router` to validate if it would consider 2 given URLs as same?